core/src/net/mod.rs

```diff
@@ -1246,6 +1311,89 @@ pub mod net_test_helpers {
                 NetworkStatus::ConnectionLost(_) => self.connection_lost += 1,
                 NetworkStatus::ConnectionDropped(_) => self.connection_dropped += 1,
                 NetworkStatus::ConnectionClosed(_) => self.connection_closed += 1,
+                NetworkStatus::BlockedSystem(sys_path) => self.blocked_systems.push(sys_path),
+                NetworkStatus::BlockedIp(ip_addr) => self.blocked_ip.push(ip_addr),
+                NetworkStatus::UnblockedSystem(sys_path) => {
+                    self.blocked_systems.retain(|s| s == &sys_path)
+                }
+                NetworkStatus::UnblockedIp(ip_addr) => self.blocked_ip.retain(|ip| ip == &ip_addr),
```

Comment:
Is this retain correct, shouldn't it be `=!` ? 
The integration tests doesn't test that the Unblocked is sent on the NetworkStatusPort so it's not actually used by the tests right now.

Reply:
Yes, will fix